tests/test_urls.py

```diff
@@ -61,11 +61,6 @@ def test_activity_page_401_if_not_logged_in(self):
         rv = self.app.get('/me')
         self.assertEqual(rv.status_code, 401)
 
-    def test_activity_page_403_view_other_user_activity(self):
```

Comment:
What's the motivation for removing this test?

(Sorry, I'm only slightly up to speed on the evolution of this issue, trying to reboot my brain)